rest/admin_api.go

```diff
@@ -150,20 +145,47 @@ func (h *handler) handleGetDbConfig() error {
 			return base.HTTPErrorf(http.StatusNotFound, "database config not found")
 		}
 
-		h.response.Header().Set("ETag", dbConfig.Version)
+		h.response.Header().Set("ETag", bucketDbConfig.Version)
+	}
+
+	// refresh_config=true forces the config loaded out of the bucket to be applied on the node
```

Comment:
Does refresh_config force reload even if the versions already match?  Or is that handled lower down.

Reply:
Nope, it will skip applying the config if the CAS from the bucket matches the one we have already loaded in-memory. This is done down inside `_applyConfig` with some debug logging.

```
[DBG] Database %q bucket %q config has not changed since last update.
```

Reply:
As discussed yesterday, pushed a commit to make `refresh_config` force a reload even if the in-memory database is up to date.